sechub-wrapper-prepare/src/main/java/com/mercedesbenz/sechub/wrapper/prepare/prepare/PrepareWrapperContext.java

```diff
@@ -1,23 +1,47 @@
 package com.mercedesbenz.sechub.wrapper.prepare.prepare;
 
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
 import com.mercedesbenz.sechub.commons.model.SecHubConfigurationModel;
+import com.mercedesbenz.sechub.commons.model.SecHubMessage;
+import com.mercedesbenz.sechub.commons.model.SecHubRemoteDataConfiguration;
 import com.mercedesbenz.sechub.wrapper.prepare.cli.PrepareWrapperEnvironment;
 
 public class PrepareWrapperContext {
 
     private SecHubConfigurationModel secHubConfiguration;
     private PrepareWrapperEnvironment environment;
+    private List<SecHubRemoteDataConfiguration> remoteDataConfigurationList = new ArrayList<>();
+    private Collection<SecHubMessage> userMessages = new ArrayList<>();
 
     public PrepareWrapperContext(SecHubConfigurationModel secHubConfiguration, PrepareWrapperEnvironment environment) {
         this.secHubConfiguration = secHubConfiguration;
         this.environment = environment;
     }
 
+    public void setRemoteDataConfigurationList(List<SecHubRemoteDataConfiguration> remoteDataConfigurationList) {
+        this.remoteDataConfigurationList = remoteDataConfigurationList;
+    }
+
+    public void addUserMessage(SecHubMessage message) {
```

Comment:
You have an add method .. okay to hide the internal access.
But later you provide a getter for the user messages...

context.getUserMessages().add(m1) or 
context.getUsermessages().add(asList(m1,m2,m3)

versus
context.addUserMessag("...")

... maybe unnecessary.